<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMain</class>
 <widget class="QMainWindow" name="AdminMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1956</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1951</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>商品管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>1491</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>502</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>增加该库存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除该商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>1160</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>510</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>折扣设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>1543</x>
        <y>90</y>
        <width>381</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>1630</x>
        <y>10</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>修改描述</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>224</x>
        <y>502</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>修改名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>500</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>1280</x>
        <y>500</y>
        <width>231</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>334</x>
        <y>12</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>订单管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>931</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>按时间升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按时间降序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按金额升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按金额降序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按用户分组</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>按用户搜索</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>时间区间搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>965</x>
        <y>90</y>
        <width>971</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>10</y>
        <width>621</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>该订单具体信息</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>用户管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>1201</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>查找用户ID</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>500</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>删除该用户</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>90</y>
        <width>501</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>性别</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>男</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>女</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>电话</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>地址</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>1560</x>
        <y>500</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>添加用户</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>10</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>管理员管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>981</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索ID</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1230</x>
        <y>100</y>
        <width>491</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>1390</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>520</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>630</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>退出程序</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>613</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>623</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
